llm/vllm/README.md

```diff
@@ -126,3 +126,61 @@ curl http://$IP:8000/v1/chat/completions \
   }
 }
 ```
+
+## Serving Mixtral 8x7b model with vLLM and SkyServe
```

Comment:
We already have the mixtral 8x7b + vLLM and SkyServe in [llm/mixtral](llm/mixtral). Should we just make the example above to be launchable with sky serve and have an additional link refering to the [llm/mixtral](llm/mixtral)?

Reply:
Make sense! Changed. PTAL again 🫡